pdfmaker/kassia.py

```diff
@@ -44,10 +45,15 @@ def setDefaults(self):
         self.pageAttrib['line_height'] = 72
         self.pageAttrib['line_width'] = self.pageAttrib['paper_size'][0] - (self.pageAttrib['left_margin'] + self.pageAttrib['right_margin'])
 
-        psalticaTTF = "fonts/EZ Psaltica.TTF"
-        oxeiaTTF    = "fonts/EZ Oxeia.ttf"
-        omegaTTF    = "fonts/EZ Omega.ttf"
+        psalticaTTF     = "fonts/EZ Psaltica.TTF"
+        specialOneTTF   = "fonts/EZ Special-I.TTF"
+        specialTwoTTF   = "fonts/EZ Special-II.TTF"
+        oxeiaTTF        = "fonts/EZ Oxeia.ttf"
+        omegaTTF        = "fonts/EZ Omega.ttf"
+
```

Comment:
This is good for now. Eventually let's move this off into a separate file that registers font and throws up error if any are missing.